studio/src/rte/rte-field.js

```diff
@@ -23,6 +23,61 @@ const isNodeCheckoutLink = (node) => {
     return node.type.name === 'link' && node.attrs['data-wcs-osi'] !== null;
 };
 
+class LinkNodeView {
```

Comment:
This is an interesting concept and can be powerful but I'm not sure if we need for links.

Reply:
The reason I added this is because we are using `atom: true` in our link nodes, and that attribute makes the node an indivisible unit meaning it does not have a built-in way to update the DOM. With this class we are overriding that behavior to update the DOM of the nodes (in this case update the `textContent` of the selection).

Reply:
https://discuss.prosemirror.net/t/nodespec-with-atom-true-and-content-inline-and-focus/6040